libImaging/Resample.c

```diff
@@ -184,7 +184,6 @@ precompute_coeffs(int inSize, int outSize, struct filter *filterp,
                 } else if (x == xmax - 1) {
                     xmax -= 1;
                 }
-                continue;
             }
             k[x] = w;
```

Comment:
> Is this bug a case of using uninitialized memory?

Yes

> Would initializing all the coefficients to 0 have prevented this?

Yes. Actually, I was certain that all memory is initialized:

1. `kk` has size of `outSize` * `kmax` doubles;
2. `xbounds` has size of two `outSize` ints;
3. We iterate `xx` from 0 to `outSize` in outer loop, so we initialize whole `xbounds`;
4. We iterate `x` in inner loop from 0 to `xmax` and initialize it with some coefficient `w`;
5. Then iterate `x` from current `x`(`xmax`) to `kmax`, and initialize it with zeroes.

So, `kk` and `xbounds` should be guaranteed initialized. The bug was on the fourth step: there was a `continue`, so on some conditions, some `kk` elements were not initialized.